build.jl

```diff
@@ -0,0 +1,31 @@
+using Images
+f = open("open.txt")
+
+content = readstring(f)
+
+f = readdir("images/")
+
+for i in f
+	println("Processing "*i)
+	img = load("images/"*i)
+	fname, ext =splitext(i)
+	file=""
+	if length(size(img))==2
+		file=fname*".png"
+		if !isfile("webimages/"*fname*".png")
```

Comment:
This could be `joinpath("webimages", file)` or `"webimages/"*file`. The first uses the official directory-separator on each platform, but I think julia performs the translation internally anyway, so I doubt it really matters.